torchio/transforms/augmentation/intensity/random_gamma.py

```diff
@@ -0,0 +1,56 @@
+from typing import Tuple, Optional, List
+import torch
+from ....torchio import DATA, TypeRangeFloat
+from ....data.subject import Subject
+from .. import RandomTransform
+
+
+class RandomGamma(RandomTransform):
+    r"""Change contrast of an image by setting its values to the power
+    :math:`\gamma`.
+
+    Args:
+        log_gamma_range: Tuple :math:`(a, b)` to compute the factor
+            :math:`\gamma` used to set the intensity to the power :math:`gamma`,
+            where :math:`\log(\gamma) \sim \mathcal{U}(a, b)`.
+            If a single value :math:`d` is provided, then
+            :math:`\log(\gamma) \sim \mathcal{U}(-d, d)`.
+        p: Probability that this transform will be applied.
+        seed: See :py:class:`~torchio.transforms.augmentation.RandomTransform`.
+        keys: See :py:class:`~torchio.transforms.Transform`.
+
+    Example:
+        >>> import torchio
+        >>> from torchio import RandomGamma, RescaleIntensity, Compose
+        >>> from torchio.datasets import FPG
+        >>> sample = FPG()
+        >>> gamma_transform = RandomGamma(log_gamma_range=(-0.3, 0.3))
+        >>> # It's better to rescale data to [0, 1] to avoid exploding values
+        >>> rescale_transform = RescaleIntensity((0, 1), (1, 99))
```

Comment:
Can you add the missing kwarg?

Reply:
Do you mean writing `rescale_transform = RescaleIntensity(out_min_max=(0, 1), percentiles=(1, 99))` instead of `rescale_transform = RescaleIntensity((0, 1), (1, 99))`? Sure.

Reply:
I meant the `percentiles` kwarg.